package-lock.json

```diff
@@ -0,0 +1,13389 @@
+{
```

Comment:
We already have yarn.lock so we should not need package-lock.json which is normally used for npm.